catalogue/webapp/pages/search/images.tsx

```diff
@@ -55,23 +55,14 @@ const ImagesSearchPage: NextPageWithLayout<Props> = ({
   imagesRouteProps,
   query,
 }): ReactElement<Props> => {
-  const { query: queryString } = query;
+  const { query: queryString = '' } = query;
```

Comment:
See other comments about using `undefined`. (I wrote this comment after the comment in `utils`, but I'm not sure which order GitHub will use.)